qutebrowser/browser/webview.py

```diff
@@ -361,6 +361,11 @@ def zoom_perc(self, perc, fuzzyval=True):
         message.info(self.win_id, "Zoom level: {}%".format(perc))
         self._default_zoom_changed = True
 
+    def setZoomFactor(self, fact):
+        self._zoom.fuzzyval = int(fact * 100)
```

Comment:
There are various places where `setZoomFactor` is used but `fuzzyval` isn't set:

- `__init__` and `on_config_changed` (should be okay as `init_neighborlist` gets called which essentially does the same)
- `zoom` (calls `zoom_perc` with `fuzzyval=False`), which is used by `:zoom-in` and `:zoom-out`.

Are you sure this won't break those calls?

(Yes, I know there should be tests for this - I'm working on that.)

Other existing places probably should get simplified.